changelog.d/1477.bugfix

```diff
@@ -0,0 +1 @@
+Make sure message diffs are not longer than the new message would be
```

Comment:
Oh, could we explain this a bit more? "Matrix message edits no longer bridge as a diff if it's longer than the new message"

Reply:
I like it. Changed in 1c526cb7